virtcontainers/pkg/oci/utils.go

```diff
@@ -347,15 +347,19 @@ func addAnnotations(ocispec specs.Spec, config *vc.SandboxConfig) error {
 
 func addAssetAnnotations(ocispec specs.Spec, config *vc.SandboxConfig) {
 	assetAnnotations := []string{
-		vcAnnotations.KernelPath,
-		vcAnnotations.ImagePath,
-		vcAnnotations.InitrdPath,
+		vcAnnotations.AssetHashType,
```

Comment:
are you not missing `vcAnnotations.HypervisorPath` here? In my local test of this PR, the hypervisor annotations still gets ignored. Upon including `vcAnnotations.HypervisorPath` here, it works.

Reply:
@gkunz - apologies - `git stash` fail :) I've actually found a way to make the code more robust, so marking as do-not-merge for now. I'll push an update tomorrow...

Reply:
@jodh-intel no problem. Thanks again for addressing the issue so quickly!